### PR TITLE
Add ID5 detection

### PR DESCRIPTION
**Pull request by max-ostapenko**
Website: https://id5.io

One of the prominent cookie-alternative audience solutions: https://blog.admixer.com/cookie-alternatives/

Examples:
1. https://adamtheautomator.com/
2. https://actu.fr/
3. https://www.freepik.com/

### CODE REVIEW

File: src/technologies/i.json [hunk 1]
```diff
@@ -31,6 +31,28 @@
...
     "website": "https://ibm.com/software/webservers/httpservers"
   },
+  "ID5": {
+    "cats": [
+      36,
+      77
+    ],
+    "description": "ID5 is the leading identity provider powering addressable advertising for brands, publishers, and their technology partners.",
+    "saas": true,
+    "icon": "ID5.png",
```
Comment (nurbek91) [inline src/technologies/i.json:41]: ID5
svg

File: src/technologies/i.json [hunk 2]
```diff
@@ -31,6 +31,28 @@
     "icon": "IBM.svg",
     "website": "https://ibm.com/software/webservers/httpservers"
   },
+  "ID5": {
+    "cats": [
+      36,
+      77
+    ],
+    "description": "ID5 is the leading identity provider powering addressable advertising for brands, publishers, and their technology partners.",
```
Comment (nurbek91) [inline src/technologies/i.json:39]: ID5 is a company that offers an identity solution for digital advertising, providing a Universal ID that enables privacy-compliant user recognition and tracking across websites without relying on personal information or third-party cookies.

File: src/technologies/i.json [hunk 3]
```diff
@@ -31,6 +31,28 @@
     "icon": "IBM.svg",
     "website": "https://ibm.com/software/webservers/httpservers"
   },
+  "ID5": {
+    "cats": [
+      36,
+      77
+    ],
```
Comment (nurbek91) [inline src/technologies/i.json:38]: maybe use one cat 67 - > Cookie compliance

File: src/technologies/i.json [hunk 4]
```diff
@@ -31,6 +31,28 @@
...
+    "description": "ID5 is the leading identity provider powering addressable advertising for brands, publishers, and their technology partners.",
+    "saas": true,
+    "icon": "ID5.png",
+    "website": "https://id5.io/",
+    "scriptSrc": [
+      "^https://(?:cdn\\.)?id5-sync\\.com",
+      "id5-api\\.js"
+    ],
```
Comment (nurbek91) [inline src/technologies/i.json:46]: `"scriptSrc": "^https://(?:cdn\\.)?id5-sync\\.com/",`

looks redundant
`"id5-api\\.js"`

File: src/technologies/i.json [hunk 5]
```diff
@@ -31,6 +31,28 @@
...
+    "xhr": ".*id5-sync\\.com",
+    "cookies": {
+      "id5": ""
+    },
+    "js": {
+      "ID5": "",
+      "__id5_instances": ""
+    }
```
Comment (nurbek91) [inline src/technologies/i.json:54]:
```
"js": {
      "ID5._version": "^(.+)$\\;version:\\1",
      "__id5_instances": ""
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!
more examples
http://cphpost.dk/
https://jubii.dk/

File: src/technologies/i.json (hunk 1 above)
Comment (max-ostapenko) [reply src/technologies/i.json:41]: @nurbek91 @AliasIO is SVG a preferred format?

File: src/technologies/i.json (hunk 3 above)
Comment (max-ostapenko) [reply src/technologies/i.json:38]: It's not compliance management, and functions as a cookie-sync alternative.
33Across is the most similar existing tech.

If you'd like to keep only one category then let's have 36.

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/i.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/i.json]: Hey @max-ostapenko , yes SVG preferred format

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**